GitUI/UserControls/FileStatusList.cs

```diff
@@ -56,6 +56,11 @@ public sealed partial class FileStatusList : GitModuleControl
         public new event EventHandler DoubleClick;
         public new event KeyEventHandler KeyDown;
         public new event EnterEventHandler Enter;
+
+        [Description("Disable showing open submodule menu items as bold")]
+        [DefaultValue(false)]
+        public bool DisableSubmoduleMenuItemBold { get; set; }
```

Comment:
Maybe?
```suggestion
        public bool OpenSubmoduleAsDefaultAction { get; set; }
```

Reply:
This only controls the 'bold' part, not default action.
I also avoided setting this to true by default (I expect false to be default for the property).
(Note that the 'bold' is not set for the standard default action, File History.)

You are commenting on a core problem in the current implementation, which is valid but not something I tried to address here.
I primarily want to clean up the order for next release and fix that items were bold when the should not be.
Can we settle on the order and fixes to the current implementation in this PR, it has enough changes already?

Going forward, I would like to remove setting for OpenSubmoduleDiffInSeparateWindow (always enable, the name is misleading now) and use DiffTool as the default doubleclick. With this PR, the first item is also the default and more often what I believe is expected (except for RevFile, better to change for consistency?).
The _openSubmoduleMenuItem could be duplicated to all forms too as well as the bold handling, but that will duplicate a number of lines of code.

Reply:
Sounds sensible. Please add a "TODO" in the code.

Reply:
Added a TODO, something should be added to the final commit message too (squash should probably be done, describing the shanges)